pegasus/sites.v3/hourofcode.com/i18n/en.yml

```diff
@@ -6758,3 +5978,87 @@
   dont_know: "I don't know"
 
   language_label: "Language"
+
+  ar: "Arabic"
+  az: "Azerbaijani"
+  bg: "Bulgarian"
+  bn: "Bangla"
+  br: "Breton"
+  bs: "Bosnian"
+  ca: "Catalan"
+  cs: "Czech"
+  cs-cz: "Czech (Czechia)"
+  da: "Danish"
+  da-dk: "Danish (Denmark)"
+  de: "German"
+  de-de: "German (Germany)"
+  el: "Greek"
+  el-gr: "Greek (Greece)"
+  en: "English"
+  en-gb: "English (United Kingdom)"
+  es: "Spanish"
+  es-es: "Spanish (Spain)"
+  es-mx: "Spanish (Mexico)"
+  et: "Estonian"
+  eu: "Basque"
+  fa: "Persian"
+  fa-ir: "Persian (Iran)"
+  fi: "Finnish"
+  fi-fi: "Finnish (Finland)"
+  fil-ph: "Filipino (Philippines)"
+  fr: "French"
+  gl: "Galician"
+  he: "Hebrew"
+  hi: "Hindi"
+  hi-in: "Hindi (India)"
+  hr: "Croatian"
+  hu: "Hungarian"
+  ia: "Interlingua (International Auxiliary Language Association)"
+  id: "Bahasa Indonesian"
+  is: "Icelandic"
+  it: "Italian"
+  it-it: "Italian (Italy)"
+  ja: "Japanese"
+  ja-jp: "Japanese (Japan)"
+  ka: "Georgian"
+  kk: "Kazakh"
+  ko: "Korean"
+  lt: "Lithuanian"
+  lv: "Latvian"
+  mk: "Macedonian"
+  mr: "Marathi"
+  ms: "Malay"
+  my: "Burmese"
+  nl: "Dutch"
+  nl-nl: "Dutch (Netherlands)"
+  nn: "Norwegian (Nynorsk)"
+  no: "Norwegian"
+  pl: "Polish"
+  pl-pl: "Polish (Poland)"
+  pt: "Portuguese"
+  pt-br: "Portuguese (Brazil)"
+  pt-pt: "Portuguese (Portugal)"
+  ro: "Romanian"
+  ru: "Russian"
+  sk: "Slovak"
+  sl: "Slovenian"
+  sq: "Albanian"
+  sr: "Serbian"
+  sv: "Swedish"
+  sv-se: "Swedish (Sweden)"
+  sw: "Kiswahili"
+  ta: "Tamil"
+  te: "Telugu"
+  th: "Thai"
+  tl: "Tagalog"
+  tr: "Turkish"
+  uk: "Ukrainian"
+  ur: "Urdu"
+  ur-in: "Urdu (India)"
+  ur-pk: "Urdu (Islamic Republic of Pakistan)"
+  vi: "Vietnamese"
+  zh: "Chinese (Simplified)"
+  zh-cn: "Chinese (Simplified, People's Republic of China)"
+  zh-hans: "Chinese (Simplified) (zh-Hans)"
+  zh-hant: "Chinese (Traditional) (zh-Hant)"
+  zh-tw: "Chinese (Traditional, Taiwan)"
```

Comment:
I'm slightly surprised we didn't already translate this list; any chance we did?

Also, is it possible to "indent" these under a parent tag to group them?  Or otherwise add a prefix explaining their context?

Reply:
Also, do we actually need to do this?  At least for our language dropdown across the site, we've been alright with showing each language in its own language...

Reply:
I can't find a thread in regards to our decision making here, but I'll pull in @dju90 here to fill in any context if I miss any. We received feedback from Jake on International that it would be more beneficial to have the languages in these fields to all be in the currently viewing locale.

I'm aware of the `cdo_languages` table in our Pegasus DB, but that doesn't seem to help at all in our goal to create translations for all languages outside of English. I didn't see any other translation files that would help in our case here either (ex. user viewing in Hebrew reading the translated version of Persian).

In regards to grouping these, I think it makes sense to add a prefix like `language_code_` or something. I need to check with Jorge if they've made significant progress on getting translations in other languages for these. If not I can add that change easily.